Cargo.toml

```diff
@@ -1,118 +1,120 @@
-	"core/inherents",
-	"core/keyring",
-	"core/keystore",
-	"core/network",
-	"core/panic-handler",
-	"core/primitives",
-	"core/rpc",
-	"core/rpc/primitives",
-	"core/rpc-servers",
-	"core/runtime-interface",
-	"core/runtime-interface/proc-macro",
-	"core/runtime-interface/test-wasm",
-	"core/serializer",
-	"core/service",
-	"core/service/test",
-	"core/session",
-	"core/sr-api",
-	"core/sr-api/proc-macro",
-	"core/sr-api/test",
-	"core/sr-arithmetic",
-	"core/sr-io",
-	"core/sr-primitives",
-	"core/sr-staking-primitives",
-	"core/sr-sandbox",
-	"core/sr-std",
-	"core/sr-version",
-	"core/state-machine",
-	"core/telemetry",
-	"core/test-client",
-	"core/test-runtime",
-	"core/test-runtime/client",
-	"core/transaction-pool",
-	"core/transaction-pool/graph",
-	"core/transaction-pool/runtime-api",
-	"core/trie",
-	"core/utils/fork-tree",
-	"core/utils/wasm-builder",
-	"core/utils/wasm-builder-runner",
-	"core/wasm-interface",
-	"srml/support",
-	"srml/support/rpc",
-	"srml/support/procedural",
-	"srml/support/procedural/tools",
-	"srml/support/procedural/tools/derive",
-	"srml/support/test",
-	"srml/authorship",
-	"srml/assets",
-	"srml/aura",
-	"srml/balances",
-	"srml/contracts",
-	"srml/contracts/rpc",
-	"srml/collective",
-	"srml/democracy",
-	"srml/elections",
-	"srml/elections-phragmen",
-	"srml/example",
-	"srml/executive",
-	"srml/finality-tracker",
-	"srml/generic-asset",
-	"srml/grandpa",
-	"srml/im-online",
-	"srml/authority-discovery",
-	"srml/indices",
-	"srml/membership",
-	"srml/metadata",
-	"srml/nicks",
-	"srml/offences",
-	"srml/randomness-collective-flip",
-	"srml/scored-pool",
-	"srml/session",
-	"srml/staking",
-	"srml/staking/reward-curve",
-	"srml/sudo",
-	"srml/system",
-	"srml/system/rpc",
-	"srml/timestamp",
-	"srml/treasury",
-	"srml/transaction-payment",
-	"srml/transaction-payment/rpc",
-	"srml/utility",
-	"srml/evm",
-	"node/cli",
-	"node/executor",
-	"node/primitives",
-	"node/rpc",
-	"node/rpc-client",
-	"node/runtime",
-	"node/testing",
-	"node-template",
-	"subkey",
-	"test-utils/chain-spec-builder",
+	"bin/node-template",
+	"bin/node/cli",
+	"bin/node/executor",
+	"bin/node/primitives",
+	"bin/node/rpc",
+	"bin/node/rpc-client",
+	"bin/node/runtime",
+	"bin/node/testing",
+	"bin/subkey",
+	"client/",
+	"client/api",
+	"client/authority-discovery",
+	"client/block-builder",
+	"client/chain-spec",
+	"client/chain-spec/derive",
+	"client/cli",
+	"client/consensus/aura",
+	"client/consensus/babe",
+	"client/consensus/pow",
+	"client/consensus/slots",
+	"client/consensus/uncles",
+	"client/db",
+	"client/executor",
+	"client/executor/runtime-test",
+	"client/finality-grandpa",
+	"client/header-metadata",
+	"client/keyring",
+	"client/keystore",
+	"client/network",
+	"client/rpc",
+	"client/rpc-servers",
+	"client/service",
+	"client/service/test",
+	"client/telemetry",
+	"client/transaction-pool",
+	"client/transaction-pool/graph",
+	"client/utils/fork-tree",
+	"client/utils/wasm-builder",
```

Comment:
Let's put these two into `build-utils` or just `build` at the root, but not into client.